server/src/services/precache.py

```diff
@@ -0,0 +1,68 @@
+from configparser import ConfigParser
```

Comment:
Seems like this file could be defunkt, but we can keep it for now. Lets drop in a comment with the link to the GH issue regarding the precaching

I have a feeling this will become necessary when we start hitting massive scale